src/citrine/seeding/find_or_create.py

```diff
@@ -147,6 +148,10 @@ def create_or_update(*, collection: Collection[CreationType], resource: Creation
         # Copy so that passed-in resource is unaffected
         new_resource = deepcopy(resource)
         new_resource.uid = old_resource.uid
+        # Locally created design workflows likely won't have a branch ID but
+        # need one to be updated.
+        if isinstance(old_resource, DesignWorkflow):
+            new_resource.branch_id = old_resource.branch_id
```

Comment:
Thanks.  If you are worried that other types will have `branch_id`s in the future, we should create a `has_branch` mixin for other classes